src/main/java/dev/mlml/command/argument/FloatArgument.java

```diff
@@ -0,0 +1,23 @@
+package dev.mlml.command.argument;
+
+public class FloatArgument extends ArgumentBase<Float> {
+    public FloatArgument(String name, String description, boolean isRequired) {
+        super(name, description, isRequired);
+    }
+
+    @Override
+    public Float parse(String input) {
+        return Float.parseFloat(input);
```

Comment:
Is there error handling/a way of preventing abuse on this?

Reply:
Fixed in new commit